docs/source/concepts/creating_an_sdk.rst

```diff
@@ -6,4 +6,4 @@ In progress
 For more details look the `Alfasim SDK project`_ that uses the ``HookMan``.
 
 
-.. _`Alfasim SDK project` : https://github.com/ESSS/alfasim-sdk
+.. _`Alfasim SDK project`: https://github.com/ESSS/alfasim-sdk
```

Comment:
--pedantic: Will you rename `Alfasim` to `ALFAsim`?

Reply:
Ahh true, will do that on a future PR. 👍